src/content/blog/latex-rendering.md

```diff
@@ -0,0 +1,113 @@
+---
+author: Alberto Perdomo
+pubDatetime: 2024-09-08T20:58:52.737Z
+title: Markdown and LaTeX
+slug: markdown-latex
+featured: false
+tags:
+  - rendering
+description: How to use LaTeX equations in your Markdown files for AstroPaper.
+---
+
+# Introduction to LaTeX Equations in Markdown
+
+This document demonstrates how to use LaTeX equations in your Markdown files for AstroPaper. LaTeX is a powerful typesetting system often used for mathematical and scientific documents.
+
+## Instructions
+
+In this section, you will find instructions on how to add support for LaTeX in your Markdown files for AstroPaper. 
+
+1. Install the necessary dependencies by running `npm install rehype-katex remark-math katex`.
+
+2. Update the Astro configuration to use the these libraries (see in **diff** format):
+
+```
```

Comment:
Plz add a file extension after ```?

Example:
```bash
```ts

```

Also, instead of diff, can you just copy and paste the updated file here? I belive it would look cleaner.

Something like this
```ts
export default defineConfig({
  // other configs
  markdown: {
    remarkPlugins: [
      remarkToc,
      rehypeKatex, // add rehypeKatex plugin
      [
        remarkCollapse,
        {
          test: "Table of contents",
        },
      ],
    ],
    // others
  },
  // other configs
});
```